packages/react-table/src/components/Table/examples/LegacyTableActions.tsx

```diff
@@ -29,18 +30,89 @@ interface Repository {
   prs: string;
   workspaces: string;
   lastCommit: string;
+  empty: React.ReactNode;
 }
 
 type ExampleType = 'actions' | 'actionResolver';
 
 export const LegacyTableActions: React.FunctionComponent = () => {
   // In real usage, this data would come from some external source like an API via props.
   const repositories: Repository[] = [
-    { name: 'a', branches: 'two', prs: '1', workspaces: 'four', lastCommit: 'five' },
-    { name: 'disable actions', branches: 'two', prs: '3', workspaces: 'four', lastCommit: 'five' },
-    { name: 'green actions', branches: 'two', prs: '4', workspaces: 'four', lastCommit: 'five' },
-    { name: 'extra action props', branches: 'two', prs: '5', workspaces: 'four', lastCommit: 'five' },
-    { name: 'blue actions', branches: 'two', prs: '6', workspaces: 'four', lastCommit: 'five' }
+    {
+      name: 'a',
+      branches: 'two',
+      prs: '1',
+      workspaces: 'four',
+      lastCommit: 'five',
+      empty: {
+        title: (
+          <TableText>
```

Comment:
I would argue that we shouldn't have any JSX or cellTransforms in the `Repository` objects here. The intention of this array is to represent PF-agnostic data that would be coming in via props in a real app (e.g. loaded from an API).